src/advanced-settings/AdvencedSettings.scss

```diff
@@ -0,0 +1,123 @@
+.setting-header__title {
+  font: normal 700 32px/36px 'Inter', sans-serif;
```

Comment:
It's better to use double quotes in SCSS, or we have another rule in the linter?

Reply:
And I prefer to have an additional variable for font-family, for simple override in the future via the `openedx-brand` package

Reply:
Great, thanks